### PR TITLE
Reverted the package and metro config file to address the SVG

### PR DESCRIPTION
**Pull request by deepakmali-dgsl**
Reverted the package and metro config file to address the SVG transfer vector issue and fixed the dynamic country language code issue

### CODE REVIEW

File: metro.config.js [hunk 1]
```diff
@@ -1,31 +1,27 @@
...
-    babelTransformerPath: require.resolve('react-native-svg-transformer/react-native'),
-    getTransformOptions: async () => ({
-      transform: {
-        experimentalImportSupport: false,
-        inlineRequires: true,
-      },
-    }),
-  },
-  resolver: {
+    resetCache:true,
```
Comment (Akhror) [inline metro.config.js:12]: Add space before true.

File: metro.config.js [hunk 2]
```diff
@@ -1,31 +1,27 @@
...
-const { assetExts, sourceExts } = defaultConfig.resolver;
 /**
  * Metro configuration
  * https://facebook.github.io/metro/docs/configuration
  *
  * @type {import('metro-config').MetroConfig}
  */
 
-const blacklist = require('metro-config/src/defaults/exclusionList');
+const blacklist = require('metro-config/src/defaults/exclusionList')
```
Comment (Akhror) [inline metro.config.js:10]: You may need ; at the end.

File: app/screens/home/SettingScreen.tsx [hunk 3]
```diff
@@ -620,13 +615,14 @@ const SettingScreen = (props: any): any => {
       setlanguage(allCountries[0]?.languages[0]);
     } else {
       console.log('Selected country for countryId is', countryId);
-      const selectedCountry: any = allCountries.find(
-        (country: any) => country.CountryID === countryId,
+      console.log('allCountries country for countryId is', allCountries);
```
Comment (Akhror) [inline app/screens/home/SettingScreen.tsx:618]: This message is confusing, let's please improve.

File: app/screens/home/SettingScreen.tsx [hunk 4]
```diff
@@ -150,20 +150,15 @@ const SettingScreen = (props: any): any => {
...
-    (state: any) => {
-     try {
-       state.selectedCountry.countries !== '' ? JSON.parse(state.selectedCountry.countries) : [];
-     } catch (error) {
-       console.error('Failed to parse countries JSON:', error);
-       return [];
-     }
-   });
+    (state: any) =>
+      state.selectedCountry.countries != '' ? JSON.parse(state.selectedCountry.countries) : [],
```
Comment (Akhror) [inline app/screens/home/SettingScreen.tsx:157]: How come selectedCountry contains multiple countries? This is against OOP concept. 
Why are we removing try/catch. How do you handle if parse method throws error?

Comment (deepakmali-dgsl) [reply app/screens/home/SettingScreen.tsx:157]: If we used try/catch its shown undefine error on settings screen that why we removed the try/catch.

File: app/screens/home/SettingScreen.tsx (hunk 3 above)
Comment (deepakmali-dgsl) [reply app/screens/home/SettingScreen.tsx:618]: This log added for testing purpose so removed now. thanks

File: metro.config.js (hunk 2 above)
Comment (deepakmali-dgsl) [reply metro.config.js:10]: ; is already there in the file but not sure why its showing in the comment. I have updated again.

File: app/screens/home/SettingScreen.tsx (hunk 4 above)
Comment (Akhror) [reply app/screens/home/SettingScreen.tsx:157]: Then let's wrap it up with if condition to make sure state.selectedCountry is defined before accessing its property.

**Review by Akhror — APPROVED** [review]

**Pull request merged**